Braintree.podspec

```diff
@@ -50,7 +50,10 @@ Pod::Spec.new do |s|
     s.source_files = "Sources/BraintreeDataCollector/*.swift"
     s.dependency "Braintree/Core"
     s.vendored_frameworks = "Frameworks/XCFrameworks/PPRiskMagnes.xcframework"
-    s.resource_bundle = { "BraintreeDataCollector_PrivacyInfo" => "Sources/BraintreeDataCollector/PrivacyInfo.xcprivacy"}
+    s.resource_bundle = {
+       "BraintreeDataCollector_PrivacyInfo" => "Sources/BraintreeDataCollector/PrivacyInfo.xcprivacy",
+       "PPRiskMagnes_PrivacyInfo" => "Frameworks/XCFrameworks/PPRiskMagnes.xcframework/ios-arm64/PPRiskMagnes.framework/PrivacyInfo.xcprivacy"
```

Comment:
This needs both the simulator & device entries

Reply:
Why? I'm not clear on how these sim and device folders are handled at different times.

Reply:
I tried including both and on simulator it produced both sets of privacy terms.
under MagnesDevice and MagnesSimulator.
I feel like this is more confusing than having duplicates for Xcode 15.3 SPM implementation

Reply:
We decided to close out the ticket and file a bug with Cocoapods.